gitlab/resource_gitlab_application_test.go

```diff
@@ -0,0 +1,133 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/xanzy/go-gitlab"
+)
+
+func TestAccGitlabApplication_basic(t *testing.T) {
+	var application gitlab.Application
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabApplicationDestroy,
+		Steps: []resource.TestStep{
+			{
```

Comment:
Can you add another TestStep with different config, to test what happens when the attributes are changed?